.github/workflows/build-test-package.yml

```diff
@@ -114,6 +127,8 @@ jobs:
   test-build:
     name: Verify Packages
 
+    if: needs.build.outputs.is_alpha == 0
```

Comment:
This is fine for now but will not work long term.  I'm wondering if this job should fail if the version is alpha.  When testing, we should be bumping the version before running a release.  That part of the process has not been written so I understand that it won't work right now.